frontend/packages/dev-console/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Web Terminal
+    As a user, I should be able to use web terminal
+
+
+Background:
+    Given user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has created namespace "aut-terminal-basic"
+    And user is at the Topolgy page
+
+
+@regression, @smoke
+Scenario: Web Terminal icon Masthead
+    Given user is at topology page
+    Then user will see the Web Terminal icon on Masthead
+
+
+@regression, @smoke
+Scenario: Web Terminal window
+    Given user is at topology page
+    When user clicks on the Web Terminal icon on the Masthead
+    Then user will see the terminal window
+
+
+@regression, @smoke
```

Comment:
Do you think this is required for smoke test suite

Reply:
Kept in regression only